packages/react-native-bottom-tabs/src/TabView.tsx

```diff
@@ -116,10 +116,13 @@ interface Props<Route extends BaseRoute> {
    */
   getTestID?: (props: { route: Route }) => string | undefined;
 
-  /**
-   * Background color of the tab bar.
-   */
-  barTintColor?: ColorValue;
+  tabBarStyle?: {
+    /**
+     * Background color of the tab bar.
+     */
+    barTintColor?: ColorValue;
```

Comment:
This should be named `backgroundColor` to follow what we have in a normal stylesheet